### ISSUE TITLE
After freechipsproject/firrtl#706, interpreter tests fail

### ISSUE THREAD

**Issue by ucbjrl** [freechipsproject/firrtl-interpreter#106]
It looks like the firrtl used in two interpreter tests needs some tweaking as a result of freechipsproject/firrtl#706:

It's curious that these are `clock` and `reset`.

```bash
[info] ModuleInLineSpec:
 ...
[info]   firrtl.passes.PassExceptions: firrtl.passes.CheckInitialization$RefNotInitializedException:  : [module Level1]  Reference level2 is not fully initialized.
[info]    : level2.reset <= VOID
[info] firrtl.passes.CheckInitialization$RefNotInitializedException:  : [module Level1]  Reference level2 is not fully initialized.
[info]    : level2.clk <= VOID
[info] firrtl.passes.CheckInitialization$RefNotInitializedException:  : [module Level2]  Reference memory is not fully initialized.
[info]    : memory.clk <= VOID
[info] firrtl.passes.CheckInitialization$RefNotInitializedException:  : [module Level2]  Reference memory is not fully initialized.
[info]    : memory.reset <= VOID
[info] firrtl.passes.CheckInitialization$RefNotInitializedException:  : [module Top]  Reference level1 is not fully initialized.
[info]    : level1.clk <= VOID
[info] firrtl.passes.CheckInitialization$RefNotInitializedException:  : [module Top]  Reference level1 is not fully initialized.
[info]    : level1.reset <= VOID
[info] firrtl.passes.PassException: 6 errors detected!
 ...
[info] VCDSpec:
[info] vcd
[info] - should be able to generate unique ids
[info] - should allow add wires
[info] - should ignore calls to wire changed when value has not changed
[info] VCD reader
[info] - should be able to read a file
[info] vcd log containing negative numbers
[info] - should work correctly and be runnable from vcd output file
[info] Using VCD output as a golden model test of a circuit
[info] - should be able to create a VCD then replay the VCD testing inputs *** FAILED ***
[info]   firrtl.passes.CheckInitialization$RefNotInitializedException: : [module VcdAdder]  Reference do_add is not fully initialized.
[info]    : do_add.clock <= VOID
[info]   at firrtl.passes.CheckInitialization$$anonfun$firrtl$passes$CheckInitialization$$checkInitM$1$2.apply(CheckInitialization.scala:83)
[info]   at firrtl.passes.CheckInitialization$$anonfun$firrtl$passes$CheckInitialization$$checkInitM$1$2.apply(CheckInitialization.scala:78)

```

**Comment by chick** [comment]
@ucbjrl I am not seeing this error.
I'm testing with current master of firrtl
 could it have gotten fixed some other way

**Comment by ucbjrl** [comment]
Still fails for me (and Jenkins).

**Comment by chick** [comment]
Dang! I'll keep looking

**Comment by chick** [comment]
@ucbjrl where do I go to see the jenkins output. I just can't reproduce. I can call that pass directly with the AST at several different points and I can't generate the error. I can't find any flags or options that would seem to be controlling the behavior.  I'm stumped at the moment.

**Comment by ucbjrl** [comment]
You should see the problem with firrtl commit 69cf8f1a084e477284d8ae48a4a7abc499e92991, firrtl-interpreter commit eac93c3d70c42b27159005c233fb49d5e5b4f4f2.
```bash
sbt clean test
```
should show the problem with the two tests `ModuleInLineSpec` and `VCDSpec`.

It's a little tricky to get to the Jenkins logs from outside Berkeley. Set up an ssh tunnel to a17.millennium.berkeley.edu mapping local port 8080 to remote port 8080:
```bash
ssh -L 8080:localhost:8080 ucbjrl@a17.millennium.berkeley.edu
```
(replace `ucbjrl` with your UC Berkeley id)

Then open `http://localhost:8080/jenkins/job/firrtl-interpreter%20no%20git%20compile/824/consoleFull` in your browser and search for `initialized`.

**Comment by chick** [comment]
If I checkout firrtl `git checkout 69cf8f1a084e477284d8ae48a4a7abc499e92991` I can't run sbt, I get 
```
dhcp-47-56:chick brew firrtl > sbt
java.lang.VerifyError: Uninitialized object exists on backward branch 487
Exception Details:
  Location:
    scala/reflect/internal/Types.appliedType(Lscala/reflect/internal/Types$Type;Lscala/collection/immutable/List;)Lscala/reflect/internal/Types$Type; @545: goto
  Reason:
    Error exists in the bytecode
```

**Comment by chick** [comment]
Appears to be fixed by PR #107
